jobs/nightly-rpm-builds.yml

```diff
@@ -26,6 +26,24 @@
 
             CENTOS_VERSION=7
 
+            CENTOS_ARCH=x86_64
+      - trigger-builds:
```

Comment:
There are yaml check errors in this PR.
I think the alignment of this block is not quite correct: This line is indented more than the other `-trigger-builds:` lines.

https://github.com/gluster/centosci/pull/136/checks?check_run_id=1872130444

https://travis-ci.com/github/gluster/centosci/builds/216666399

Reply:
Will resolve the same.